lib/decoder.js

```diff
@@ -0,0 +1,48 @@
+
+
+export default function Decoder() {
```

Comment:
I'm fine if we don't want to expose this, although I suspect for source-map parity it will eventually be useful to do so. But we can wait.